aten/src/ATen/record_function.cpp

```diff
@@ -277,10 +277,12 @@ class CallbackManager {
       bool is_start) {
     try {
       if (is_start) {
-        ctx = rfcb.start()(rf);
+        ctx = rfcb.start() ? rfcb.start()(rf) : nullptr;
```

Comment:
How about something like?
```suggestion
        if (auto& start = rcbf.start()) ctx = start(rf) else ctx =  nullptr;
```

Reply:
there's no reason to complicate the code like this; start() is an inlineable load of a single pointer